services/licenses.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+
+function normalizeTemplate(template) {
+  const { title } = template;
+  return title.replace(/^Template:/, '');
+}
+
+function formatPageTemplates(response) {
+  const { pages } = response;
+  assert.ok(pages, 'notFound');
+  const { templates = [] } = Object.values(pages)[0];
```

Comment:
reading this, I am a little concerned that we don't handle pages well. It seems that the wikimedia/pedia APIs return paged results and we only ever read the first page. What if there are more templates and we need to read all pages?

`Lizenzhinweisgenerator/js/app/Api.js` (in `_query()`) handles paged answers (which may need a little refactoring to be more readable/straight forward). Shouldn't we also handle paged answers?